tests/components/homekit/test_img_util.py

```diff
@@ -0,0 +1,62 @@
+"""Test HomeKit img_util module."""
+from homeassistant.components.camera import Image
+from homeassistant.components.homekit.img_util import (
+    TurboJPEGSingleton,
+    scale_jpeg_camera_image,
+)
+
+from .common import EMPTY_8_6_JPEG, mock_turbo_jpeg
+
+from tests.async_mock import patch
+
+EMPTY_16_12_JPEG = b"empty_16_12"
+
+
+def test_turbojpeg_singleton():
+    """Verify the instance always gives back the same."""
+    assert TurboJPEGSingleton.instance() == TurboJPEGSingleton.instance()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)